### ISSUE TITLE
Translate authorize request

### ISSUE THREAD

**Issue by EKE00372** [ShikuAlexei/Guidelime_Shiku#1]
Hello, can i get a authorize for translate this guide? target locale is zhTW/zhCN.

even grant a different language version public on curse or just a fork on github.

**Comment by ShikuAlexei** [comment]
Yes friend, if you just need permission go ahead, if there is something I have to do on my end to grant permission let me know.

**Comment by EKE00372** [comment]
Thank you very much!

**Comment by EKE00372** [comment]
These days I've do zhTW translate ~ level 42.

zhTW branch:
https://github.com/EKE00372/Guidelime_Shiku/tree/zhTW

Please tell me if there're anything should be changed. Thank you!
